coffee_github/src/repository.rs

```diff
@@ -57,6 +59,7 @@ impl Github {
             url: url.clone(),
             plugins: vec![],
             branch: "".to_owned(),
+            commit: "".to_owned(),
```

Comment:
Mh btw this is wrong for this PR I think? we should store the commit inside the plugin, we already know the commit of the repository we are using git for this reason